custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -449,13 +450,45 @@ class SchemaValidator:
     actual validation on the respective dataframes.
     """
 
-    def __init__(self, type_schema: dict, strict=True, verbose=False):
+    _DEFAULT_TYPE_SCHEMA_CODEDIR_PATH = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), "..", "resource", "default_typeschema")
+    )
```

Comment:
There may be weirdness building this into a wheelfile.

Reply:
There isn't. This is how other resource files are referenced